x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -880,4 +809,24 @@ public static boolean isPlatinumOrTrialOperationMode(final OperationMode operati
     public synchronized XPackLicenseState copyCurrentLicenseState() {
         return new XPackLicenseState(this);
     }
+
+    private synchronized boolean checkSecurityEnabled() {
```

Comment:
It is possible to replace this method with just a call of `checkMinimumLicense(MISSING, true, false, true)`. But I feel a separate method clears the intention and is more readable.

Reply:
> It is possible to replace this method with just a call of checkMinimumLicense(MISSING, true, false, true)

is it ? That wouldn't handle the intricacies of `isSecurityExplicitlyEnabled` vs `isSecurityEnabled`

Reply:
`isSecurityEnabled` is called inside `checkMinimumLicense` as well. So it can handle the subtlety. I do prefer to keep it separated for clarity.